testcases/kernel/syscalls/fanotify/fanotify01.c

```diff
@@ -285,7 +285,8 @@ void test01(void)
 		 * invalidate it so that we don't check it again
 		 * unnecessarily
 		 */
-		close(event->fd);
+		if (event->fd >= 0)
+			close(event->fd);
```

Comment:
I guess that we can make use of SAFE_CLOSE() here that would check the return value from the close() as well, since we are sure that we attempt to close only valid file descriptors after the change.